microsoft-outlook/latest/install/index.md

```diff
@@ -124,7 +122,76 @@ Ensure that you have applied the Alfresco Outlook Server AMP files ([see Install
 
     >**Note:** If you added a client license, the license key is displayed, with a message to check the **Alfresco Client > Configure > License** tab in Microsoft Outlook (do this check after you have installed Alfresco Outlook Client).
 
-### Install the Alfresco Outlook Client in Microsoft Outlook
+## Install Transform Engine
+
+The Outlook Integration Transform Engine (or T-Engine) enables transformation of MSG and EML files into PDF format when used with the Alfresco Transform Service. The Outlook Integration T-Engine is available both as a Docker container and as a Web Archive (WAR) file.
+
+
+**TODO:**
+* *Add more info about the Docker container and WAR file use cases?*
+* *Separate "Install AMPs" + "Install T-Engine on Tomcat" from "Install using Docker"?*
+* *Add "Containerized deployment" sub-section?*
+ 
+
+### Install T-Engine on Tomcat
+
+If you wish to use an existing Tomcat application server, you can use the WAR bundle to install the Outlook Integration T-Engine.
+
+1. Install the latest version of [Tomcat 8.5](https://tomcat.apache.org/download-80.cgi){:target="_blank"} to run as a service.
+
+    See the [Tomcat documentation](https://tomcat.apache.org/tomcat-8.5-doc/index.html){:target="_blank"} for more information about configuring and using a Tomcat server.
+
+    For information about securing Tomcat, see [Tomcat security considerations](https://tomcat.apache.org/tomcat-8.5-doc/security-howto.html).
+
+2. Copy the WAR file into your `<TOMCAT_HOME>/webapps` folder.
+
+    The WAR (`transform-outlook-webapp-1.0.0.war`) is included in the distribution zip.
+
```

Comment:
Add here:
To enable ActiveMQ in Outlook T-Engine set ActiveMQ url property in JAVA_OPTS variable:
-Dactivemq.url=${activemq.url}
To enable shared file store in Outlook T-Engine set shared file store url property in JAVA_OPTS variable:
-DfileStoreUrl=${shared.file.store.url}

Reply:
@swall-developer Thanks for the review and additional content - this has now been added in the latest commit (slightly reworded)